### DISCUSSION TITLE
bat could have been called rat

### DISCUSSION THREAD

**Discussion by oredaze** [sharkdp/bat#2470] [General]
I am sorry for the pointless topic, but I could not help myself.

Think about it, cat + rust = rat. Also you can think of it like the program rats out the contents of the file.

What a wasted opportunity :smile: 
Or maybe you thought of that idea already?
I am just a bit bummed out about it, no big deal. Have a nice day :+1:

**Comment by spiercy** [comment]
Agreed. I searched the discussions just to make sure I wasn't the first to make this point. Since cat has long been sourced in C (c. 1979), one would think a port to Rust would've been more likely called rat. To have called it bat, it probably ought to have been written in B. Just an opinion of course. Love the features.
